### PR TITLE
Update appdata

### PR DESCRIPTION
**Pull request by oscfdezdz**
# Description
Adds some more information to appdata and reduces the content rating lines as they are all "none" which is the default value.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- [X] Following [documentation](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html) and GNOME Software 41 upcoming changes.

### CODE REVIEW

**Comment by mirkobrombin** [comment]
Thanks! Looks good to me.

File: data/com.usebottles.bottles.appdata.xml.in [hunk 1]
```diff
@@ -60,38 +60,21 @@
...
     <url type="help">https://docs.usebottles.com</url>
+    <url type="donation">https://usebottles.com/funding</url>
+    <url type="translate">https://hosted.weblate.org/engage/bottles</url>
+    <recommends>
+      <control>pointing</control>
+      <control>keyboard</control>
+      <control>touch</control>
+    </recommends>
+    <requires>
+      <display_length compare="ge">medium</display_length>
```
Comment (oscfdezdz) [inline data/com.usebottles.bottles.appdata.xml.in:76]:
```suggestion
      <display_length compare="ge">768</display_length>
```

@mirkobrombin I've read [this post](https://blogs.gnome.org/tbernard/2021/09/07/ready-for-software-41/) and as it points out, display_length size constants are broken, so it should be changed to its numerical equivalent for it to work properly. Sorry for the inconvenience 😅

Comment (mirkobrombin) [reply data/com.usebottles.bottles.appdata.xml.in:76]: Oh no problem. Can you make another PR? If you can't, I'll fix it as soon as I can

Comment (oscfdezdz) [reply data/com.usebottles.bottles.appdata.xml.in:76]: https://github.com/bottlesdevs/Bottles/pull/525 fixes it

**Pull request merged**